app/controllers/dashboard_controller.rb

```diff
@@ -2,5 +2,8 @@ class DashboardController < ApplicationController
   before_action :authorize!
 
   def show
```

Comment:
This is exactly what I was going to comment on! Great that you've been able to point out your next steps.